packages/theme/package.json

```diff
@@ -15,7 +15,8 @@
   "type": "module",
   "files": [
     "/dist",
-    "/oclif.manifest.json"
+    "/oclif.manifest.json",
+    "/theme-cli"
```

Comment:
This is including all the files there in the package, right? What's the difference in size?

What about being a bit more explicit to avoid the test folders for example? Another alternative is using `.npmignore` (more info [here](https://blog.npmjs.org/post/165769683050/publishing-what-you-mean-to-publish)).